bfe-verso-profiles/profiles/BIBFRAME 2.0 Admin Metadata.json

```diff
@@ -735,8 +735,6 @@
     "title": "BIBFRAME 2.0 Admin Metadata",
     "description": "Metadata for BIBFRAME Resources",
     "date": "2018-10-24",
-    "author": "NDMSO",
-    "adherence": "BIBFRAME 2.0",
-    "source": ""
```

Comment:
I could add `source` and `adherence` properties to the schema and not require them.  That approach didn't occur to me until I had already deleted them.